scalafix/output/src/main/scala/fix/BreakoutSrc.scala

```diff
@@ -58,7 +58,7 @@ class BreakoutSrc(ts: Iterable[Int], vec: Vector[Int], list: List[Int], seq: Seq
   (ts.iterator ++ ts ).to(scala.collection.immutable.Set): Set[Int]
 
   // `TraversableLike ++:`
-  (ts ++: ts.view).to(scala.collection.immutable.Set): Set[Int]
+  // (ts ++: ts)(breakOut): Set[Int]
```

Comment:
This does not compile in 2.13.0-M4.

```scala
value ++: is not a member of scala.collection.View[Int]
```
I think it will be solved via this PR: https://github.com/scala/scala/pull/6674

Reply:
I see, that’s because we don’t have `++:` on `Iterable`.